python/ray/serve/utils.py

```diff
@@ -9,37 +9,70 @@
 from typing import List
 import io
 import os
+from ray.serve.exceptions import RayServeException
 
 import requests
+import pydantic
+from werkzeug.datastructures import ImmutableMultiDict
 
 import ray
 from ray.serve.constants import HTTP_PROXY_TIMEOUT
-from ray.serve.context import FakeFlaskRequest, TaskContext
+from ray.serve.context import TaskContext
 from ray.serve.http_util import build_flask_request
 import numpy as np
 
-try:
-    import pydantic
-except ImportError:
-    pydantic = None
-
 ACTOR_FAILURE_RETRY_TIMEOUT_S = 60
 
 
+class ServeRequest:
```

Comment:
Please make sure to document this (and all of the properties) and add it to the package ref